contentful_rails.gemspec

```diff
@@ -7,8 +7,9 @@ require "contentful_rails/version"
 Gem::Specification.new do |s|
   s.name        = "contentful_rails"
   s.version     = ContentfulRails::VERSION
-  s.authors     = ["Error Creative Studio"]
-  s.email       = ["hosting@errorstudio.co.uk"]
+  s.authors     = ['Contentful GmbH (David Litvak Bruno)', 'Error Creative Studio']
+  s.email       = ['david.litvak@contentful.com', 'hosting@errorstudio.co.uk']
+  s.homepage    = 'https://github.com/contentful/contentful_model'
```

Comment:
can you have more then one homepage defined like that or would you need to create a list using '[foo,bar]'? (See next line)